components/Tooltip.tsx

```diff
@@ -1,15 +1,27 @@
+import Image from "next/image";
+import { useState } from "react";
+
 export default function Tooltip() {
-  return (
-    <div className="flex min-h-[59px] w-[100%] items-center justify-start rounded-[3px] bg-[#3284ff] pl-[20px] md:mb-[25px] md:ml-[20px] md:mt-[50px] md:w-[96%]">
-      <p>
+  const [hidden, setHidden] = useState(false);
+
+  const removeTooltip = () => {
+    setHidden(true);
+  };
+
+  return hidden ? null : (
+    <div className="flex min-h-[59px] w-[100%] items-center justify-between rounded-[3px] bg-[#3284ff] pl-[20px] md:mb-[25px] md:ml-[20px] md:mt-[50px] md:w-[96%]">
+      <div className="flex flex-col">
         <span className="text-xs font-bold leading-normal text-white">
           INSTRUCTIONS
         </span>
-        <span className="flex text-xs font-normal leading-normal text-white">
+        <span className="text-xs font-normal leading-normal text-white">
           To make modifications to your AI policy, try clicking on a section to
           start editing the content.
         </span>
-      </p>
+      </div>
+      <button className="mb-4 p-2 text-white" onClick={removeTooltip}>
+        <Image src="/images/x.svg" alt="x icon" width={20} height={20} />
```

Comment:
let's use an actual x image from the library `@radix-ui/react-icons`

you can see an example in our own codebase here:
https://github.com/coursepolicy/policy-generator/blob/8ef1389809ef0123a95f5c38f16f2fdcfe19954e/components/ui/button.tsx#L6-L7

Reply:
curently it doesn't look like the design because it looks very pixelated. You can see for yourself using [staging.coursepolicy.ai](https://staging.coursepolicy.ai/policy/sample):

<img width="944" alt="Screenshot 2024-06-16 at 12 16 16 AM" src="https://github.com/coursepolicy/policy-generator/assets/8962594/2a9d67f8-fa38-4ef4-8e78-f8418a4ed4d2">